prow/spyglass/lenses/junit/lens.go

```diff
@@ -136,48 +178,78 @@ func (lens Lens) Body(artifacts []lenses.Artifact, resourceDir string, data stri
 	}
 	sort.Slice(results, func(i, j int) bool { return results[i].path < results[j].path })
 
-	jvd := struct {
-		NumTests int
-		Passed   []TestResult
-		Failed   []TestResult
-		Skipped  []TestResult
-	}{}
+	jvd := JVD{}
```

Comment:
Prefer `var jvd JVD` unless you are initializing something

https://dave.cheney.net/practical-go/presentations/qcon-china.html#_use_a_consistent_declaration_style

Reply:
done